playground/src/app.css

```
@@ -1,5 +1,11 @@
 @import "tailwindcss";
 
+@plugin "daisyui" {
+  themes: business --default;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)